### ISSUE TITLE
GAN code to convert synthetic images to real.. i.e. working with paired images

### ISSUE THREAD

**Issue by rohitsaluja22** [Zardinality/WGAN-tensorflow#12]
Hi, please share a WGAN code to convert synthetic images to real (with paired images) if you know. Any other vanila GAN would also be helpful.

**Comment by Zardinality** [comment]
@rohitsaluja22 Have you learned about [pix2pix](https://phillipi.github.io/pix2pix/) and its follow-up work [cycleGAN](https://junyanz.github.io/CycleGAN/)? You can find the code in their respected project page.

**Comment by rohitsaluja22** [comment]
@Zardinality, I want a simple model with good loss function. In the end, I want samples from real distribution only. These models which you shared I guess give a style transferred image of synthetic version since they try to reconstruct the synthetic image back.

Hope you understand what I mean. I want to try out extreme changes to the synthetic image and am not interested in reconstructing it back.. e.g. would be changing a word image in one font to the same word in another font..

**Comment by Zardinality** [comment]
@rohitsaluja22 I am still not sure what you mean exactly, but if you want a one-sided version of Cycle-GAN, here is a [repo](https://github.com/sagiebenaim/DistanceGAN).

**Comment by rohitsaluja22** [comment]
@Zardinality  check this: https://arxiv.org/pdf/1801.07156.pdf
I want to check out if vanila GAN can do this..
